utils/scripts/gen_tflite_models.py

```diff
@@ -47,6 +47,12 @@
 OUT_DIR = os.path.join(os.path.dirname(__file__), "tflite_models")
 
 
+# Generate quantization data based on the input tensor
+representative_images = np.random.randn(1,10).astype(np.float32)
```

Comment:
Does the quantization mechanism work for models with multiple inputs/ouputs (e.g. add/sub)?

Reply:
This was not my intention to do as I don't needed - at the moment, the quantization mechanism doesn't break anything - it just targets unary ops. However, I added a bit more to this for completness. Added add_int8 model gen. and tested that it works.